webapp/views.py

```diff
@@ -1229,3 +1229,9 @@ def cube_microcerts():
     }
 
     return flask.render_template("cube/microcerts.html", **data)
+
+
+def cube_home():
+    data = {}
```

Comment:
You can remove it from here and from the function call bellow, unless you will need this in the future?

Reply:
I was thinking I will need the badge icon data in the future, but I could always just hard code them from the asset server now I think about it

Reply:
Has you prefer, I was just wondering, happy to keep reviewing without changes.

Reply:
Ok thanks, I'll leave it there for now and then remove it if I end up not needing it